modules/fb_instant_articles_display/includes/InstantArticleNodeWrapper.inc

```diff
@@ -0,0 +1,547 @@
+<?php
+
+/**
+ * @file
+ * Defines the InstantArticleNodeWrapper class.
+ */
+
```

Comment:
Actually let's save the class namespace for the follow-up PR you and I talked about (after this one is merged). So… never mind this comment for now :smile: